automated_analysis.py

```diff
@@ -174,36 +173,57 @@
             for code in cc.code_scheme.codes:
                 if code.control_code == Codes.STOP:
                     continue
-                demographic_distributions[cc.analysis_file_key][code.string_value] = 0
+                demographic_distributions[cc.analysis_file_key][code.code_id] = 0
+            total_relevant[cc.analysis_file_key] = 0
 
     for ind in individuals:
         if ind["consent_withdrawn"] == Codes.TRUE:
             continue
 
-        for plan in PipelineConfiguration.SURVEY_CODING_PLANS:
+        for plan in PipelineConfiguration.DEMOG_CODING_PLANS:
             for cc in plan.coding_configurations:
-                if cc.include_in_theme_distribution == Codes.FALSE:
```

Comment:
check with RDA which demog variables they need in the theme distribution file. It could be part of the analysis files but not necessarily needed in theme distribution.

Reply:
thanks, updated